pages/categories/index.js

```diff
@@ -1,13 +1,36 @@
 import useSWR from "swr";
+import SubCatCard from "~/components/SubCatCard";
 
 const Categories = () => {
-  const { data } = useSWR("/api/categories");
+  const { data: categories } = useSWR("/api/issues");
   return (
     <div>
-      <h2>Categories to be shown here</h2>
-      {data
-        ? data.data.map((item, index) => <li key={index}>{item.name}</li>)
-        : null}
+      <div className="flex flex-col md:flex-row md:flex-wrap justify-end items-center md:justify-between mb-20">
+        {categories &&
+          categories.data.map(({ id, category_id, name }) => (
+            <SubCatCard
+              key={id}
+              backgroundColor={
+                category_id === 1
+                  ? "red"
+                  : category_id === 2
+                  ? "pink"
+                  : category_id === 3
+                  ? "purple"
+                  : "blue"
+              }
+              title={name}
+              category={
+                category_id === 1
```

Comment:
Can we make these dynamic? Maybe we could do another api call for the categories and use that to check for the name

Reply:
I will give it a try, Thank you